federation-js/src/composition/compose.ts

```diff
@@ -120,9 +120,6 @@ export interface KeyDirectivesMap {
  */
 type ValueTypes = Set<string>;
 
-export type ComposedGraphQLSchema = GraphQLSchema & {
```

Comment:
This type is no longer needed since the `schema` we're constructing in memory no longer needs the `extensions` field with the service info.